app/src/main/java/travelbook/android/app/data/TravelBookDatabaseOpenHelper.java

```diff
@@ -0,0 +1,79 @@
+package travelbook.android.app.data;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.security.PrivateKey;
+
+/**
+ * A javadoc for working with the database.
```

Comment:
MemoriesContract is neither a javadoc nor a class for forking with the database.